ui/src/app/utils/runtime-utils.tsx

```diff
@@ -213,6 +213,10 @@ const useRuntime = (currentWorkspaceNamespace) => {
   // No cleanup is being handled at the moment.
   // When the user initiates a runtime change we want that change to take place even if they navigate away
   useEffect(() => {
+    if (!currentWorkspaceNamespace) {
+      return;
+    }
+
```

Comment:
I guess one thing to look at later (or we can try now), is whether the second param to `useEffect` is still correct here. Possibly that should be `currentWorkspaceNamespace`

Reply:
sure. I think throwing it in is safe. a new value for `currentWorkspaceNamespace` should trigger a new get runtime call.

Although, I don't think it matters at the moment since the help-sidebar is remounting on every page load and running this regardless.